examples/end-to-end-examples/support_chatbot/mcli_yamls/deploy_llm.yaml

```diff
@@ -0,0 +1,49 @@
+name: mpt-30b-composer-finetuned
+
+# Deployment configuration
+# For 30B model deployed with bf16 datatype, the model needs atleast ~60GB of memory.
+# So should be hosted on atleast 2x A100-40G or 4x A10s
+compute:
+  cluster: r7z19
+  gpus: 4
+  instance: oci.bm.gpu.a10.4
+replicas: 1 # Number of replicas to use for this run
+
+
+integrations:
+# Clone and install the examples repo so we can use the deployment helper from it
+- integration_type: git_repo
+  git_repo: KuuCi/examples
+  git_branch: support-bot
+  ssh_clone: false
+# Clone the llm-foundry repo so we can use the HF to FasterTransformer convert script from it
+- git_commit: 2c92faa5ce31888214bdb582ac7f5756d0d3dacd
+  git_repo: mosaicml/llm-foundry
+  integration_type: git_repo
+  ssh_clone: false
+
+# Add the examples folder to the PYTHONPATH so we can import the deployment helper
+# Install composer to use the cloud download helper
+command: |
+  export PYTHONPATH=$PYTHONPATH:/code/llm-foundry:/code/examples:
+  pip uninstall packaging -y
+  rm /usr/lib/python3/dist-packages/packaging-23.1.dist-info/REQUESTED
+  pip install composer[streaming,libcloud,oci]==0.14.1
+  pip install packaging==23.1
+  pip install langchain==0.0.205
```

Comment:
if you can get rid of the __all__ to avoid these installs that would be good. otherwise we can restructure a bit.